src/panels/config/integrations/integration-panels/bluetooth/bluetooth-config-dashboard.ts

```diff
@@ -0,0 +1,107 @@
+import "@material/mwc-button";
+import type { CSSResultGroup, TemplateResult } from "lit";
+import { css, html, LitElement } from "lit";
+import { customElement, property } from "lit/decorators";
+import "../../../../../components/ha-card";
+import "../../../../../components/ha-code-editor";
+import "../../../../../components/ha-formfield";
+import "../../../../../components/ha-switch";
+import { getConfigEntries } from "../../../../../data/config_entries";
+import { showOptionsFlowDialog } from "../../../../../dialogs/config-flow/show-dialog-options-flow";
+import "../../../../../layouts/hass-subpage";
+import { haStyle } from "../../../../../resources/styles";
+import type { HomeAssistant } from "../../../../../types";
+
+@customElement("bluetooth-config-dashboard")
+export class BluetoothConfigDashboard extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ type: Boolean }) public narrow = false;
+
+  protected render(): TemplateResult {
+    return html`
+      <hass-subpage .narrow=${this.narrow} .hass=${this.hass}>
+        <div class="content">
+          <ha-card
+            .header=${this.hass.localize(
+              "ui.panel.config.bluetooth.settings_title"
+            )}
+          >
+            <div class="card-actions">
+              <mwc-button @click=${this._openOptionFlow}
+                >${this.hass.localize(
+                  "ui.panel.config.bluetooth.option_flow"
+                )}</mwc-button
+              >
+            </div>
+          </ha-card>
```

Comment:
That's a bit up to you, but if you want people to end up here during troubleshooting, which was a use-case I had in mind, then we don't need to show this card. (You can visit /config/bluetooth without providing a config flow, where /config/bluetooth will act as a bluetooth hub.)

Reply:
Is there a way to reach it without manually typing in the URL.  If they have to type it in it's much more likely we will never get them to do it and the issue report will never get solved.  We have such a hard time getting debug logs and diagnostics, I want it to be as frictionless as possible to get here.

Reply:
https://my.home-assistant.io/ links can be added to the documentation page.

Reply:
I think that would limit discoverability as it would make it a lot harder to find. I have limited success getting people to click my links when working issues (well reading the whole Bluetooth docs as well)

Reply:
The only place where they are being taken to the integration panel, is through the configure button on the config flow. What would be the ultimate solution?

Reply:
> The only place where they are being taken to the integration panel, is through the configure button on the config flow. What would be the ultimate solution?

I'd love to make it even easier to discover, but I'm not sure where we could put it where they would be more likely to find it, or if its even possible with the current design.

Reply:
Another solution, is to register it as a frontend panel in the bluetooth integration. That will let it show up in the sidebar.

Reply:
That would certainly make it easier to discover 😄  Do we have precedent for doing that for any other integrations?

Reply:
[KNX](https://github.com/home-assistant/core/blob/412636a198730a75101f6c7de4f2276edb83ffad/homeassistant/components/knx/websocket.py#L71-L72) is the best example that comes to mind first. 
Quick search brings Insteon, Dynalite and lcn as well.

Reply:
As a side note, since it's a built-in panel, you may need to use frontend.async_register_built_in_panel instead of panel_custom.async_register_panel.

Reply:
I'm a bit on the fence since there's 100s of thousands of people who are suddenly gonna get a Bluetooth panel.  It's a really tough call because we wanna make it easy for them to find their devices, but we also don't wanna overwhelm them with information.

Reply:
The other API's I'm planning on adding our config entry/adapter specific so we would still need a screen for each adapter.

Reply:
having just a URL will make it too hard to be discoverable. I suggest we do use the "configure" from the config entry to go to the bluetooth panel, as we do for Zigbee, Z-Wave, Matter and KNX. And then on the bluetooth panel, have a button to open the options flow for that config entry, that way no options are lost.